presto-docs/src/main/sphinx/connector/deltalake.rst

```diff
@@ -43,9 +43,8 @@ Property Name                                   Description
                                                 ``true``.
 =============================================== ========================================================= ============
 
-Delta Lake connector reuses many of the modules existing in Hive connector.
-Modules for connectivity and security such as S3, Azure Data Lake, Glue metastore etc.
-So the configurations for these modules is same those available in Hive connector documentation.
+Delta Lake connector reuses many of the modules existing in Hive connector including modules for connectivity and security such as S3, Azure Data Lake, Glue metastore, etc.  The configurations for these modules are the same as those available in the `Hive connector <https://github.com/delta-io/connectors/#hive-connector>`_  documentation.
```

Comment:
The multiline spacing is intention to keep the documents to be readable. Line breaks in rst generally leads to spaces in the final document - so lets continue to keep around 80-100 characters/line with the new wording.